src/form.js

```diff
@@ -983,6 +983,21 @@ define([
             var _this = this;
             this._logicManager.forEachBrokenReference(updateReferences);
         },
+        isReferencedByOtherMugs: function (mug, except) {
```

Comment:
Are you proposing that the caller would reach through the form to get to the logic manager? Like `form._logicManager.isReferencedByOtherMugs`? That breaks encapsulation and makes the caller have to know things like the logic manager mainly deals with ufids rather than mugs in many cases.